contents/newsletter/first-time-founders.md

```diff
@@ -0,0 +1,131 @@
+
+They persevered for a while in the hope of turning things around, but it never worked out.
+
+### What Taylor learned
+
+Initial interest and orders convinced Taylor they were onto something, but they made a few classic first-time mistakes:
+
+1. **Working on a problem they didn't know well:** While Taylor and his co-founders knew there was a problem to solve, it wasn't one they had personally encountered or tried to solve. In contrast, when Taylor and Ev Kontsevoy founded Mailgun, "Ev was solving his own problem that he understood deeply." They quickly found product-market fit and Mailgun was acquired after just two years.
+
+2. **Talking to the wrong people:** Managers have opinions, but they rarely understand the challenges of their employees. By working with managers as design partners, rather than the people who would use the app, Profista created a solution that didn't survive contact with reality.
+
+3. **Building a solution that isn't 10x better:** Points one and two inevitably lead here, as Taylor explains:
+
+> "We were very descriptive about the solution we were building, and in the end, it wasn't appreciably better than the email and spreadsheets based workflow they were using. People don't like change, and they won't adopt a new way of doing things unless it is massively better."
+
+## More first-time founder mistakes
+
+### Not solving a real problem
+
+Exhibit A: [Humane AI Pin](https://www.theverge.com/24126502/humane-ai-pin-review)
+
+### The wrong co-founder(s)
+
+"[Companies are mostly about people](https://newsletter.posthog.com/p/what-we-learned-about-hiring-from)" and that starts with your co-founders. Marty Kausus, co-founder and CEO at [Pylon](https://usepylon.com), quit his second startup the moment he found "the right co-founders" to start a more successful venture.
+
+Marty and his co-founders got together and "pivoted weekly" for around two months before landing on Pylon, a Zendesk and Intercom alternative for managing customer operations via Slack. A year and a half later, they already have strong product-market fit.
+
+> "My biggest takeaway on co-founders is to make sure they have similar hustle, spirit, and desired lifestyle. For Pylon, we wanted to live together and live and breathe the company. We joke that we don't have "work-life balance", we have "work-life integration" and that's ideal for us, while most people would find that miserable."
+
+### Not using analytics
+
+YC's Michael Seibel says that, even now, "we see this time and time again." See also: [What engineers get wrong about analytics](https://newsletter.posthog.com/p/what-engineers-get-wrong-about-analytics).
+
+### Just shipping what users say they want
+
+Talking to customers is important, but it can go too far. This was a lesson James Evans, now co-founder at CEO at [CommandBar](https://www.commandbar.com), learned in his first startup:
```

Comment:
```suggestion
Talking to customers is important, but it can go too far. This was a lesson [James Evans](https://www.linkedin.com/in/james-evans-7086b3126/), now co-founder at CEO at [CommandBar](https://www.commandbar.com), learned in his first startup:
```